test/import.jl

```diff
@@ -193,6 +193,31 @@ using Legolas: validate, Schema, read
         end
     end
 
+    @testset "duplicate sensor_type" begin
+        rng = MersenneTwister(1234)
```

Comment:
Better to use StableRNG.jl so this works consistently on different Julia versions